lib/rules/operator-linebreak.js

```diff
@@ -63,7 +64,7 @@ module.exports = function(context) {
 
             return;
 
-        } else if (!astUtils.isTokenOnSameLine(leftToken, operatorToken) &&
+        } else if (operatorStyleOverride !== "ignore" && !astUtils.isTokenOnSameLine(leftToken, operatorToken) &&
```

Comment:
@btmills any idea why tests didn't fail when "ignore" was used without updating the schema?